internal/operators/lvm/lvm_operator.go

```diff
@@ -118,14 +120,36 @@ func (o *operator) ValidateHost(ctx context.Context, cluster *common.Cluster, ho
 		return api.ValidationResult{Status: api.Failure, ValidationId: o.GetHostValidationID(), Reasons: []string{message}}, err
 	}
 
-	// GetValidDiskCount counts the total number of valid disks in each host and return an error if we don't have the disk of required size
 	diskCount := o.getValidDiskCount(inventory.Disks, host.InstallationDiskID)
+
+	role := common.GetEffectiveRole(host)
+	message := "Logical Volume Manager requires at least one non-installation HDD/SSD disk on the host"
+	if (role == models.HostRoleWorker && !*cluster.SchedulableMasters) || *cluster.SchedulableMasters {
```

Comment:
If the host is a worker, you want to validate it regardless of schedulable masters
```suggestion
	if (role == models.HostRoleWorker || *cluster.SchedulableMasters {
```

Reply:
Right, so make it clearer with my code suggestion - either worker or schedulablemasters.

Reply:
Also can cluster.SchedulableMasters be nil?

Reply:
changed to the suggested and moved to line 109